internal/pkg/filesystem/knownpaths/paths.go

```diff
@@ -0,0 +1,309 @@
+package knownpaths
```

Comment:
Presunuty subor `model/pathstate.go` -> `knownpaths/paths.go`, .. github to zobrazuje ako novy subor